packages/ui/components/NotificationBanner/NotificationBanner.spec.tsx

```diff
@@ -61,3 +61,17 @@ describe('NotificationBanner Component', () => {
     })
   })
 })
+
+test('renders rich text when richText prop is true', () => {
+  render(
+    <NotificationBanner richText heading="Test Heading">
+      <p>Test Content</p>
+    </NotificationBanner>
+  )
+  expect(screen.getByText('Test Content')).toBeInTheDocument()
```

Comment:
`getByText` can be destructured from the render method, ie:

` const { getByText } = render(<NotificationBanner richText heading="Test Heading"><p>Test Content</p></NotificationBanner>)` then just use `getByText` directly